2-mandatory/1-weather-report.js

```diff
@@ -12,52 +12,55 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+  temp = []; //initialise an array
```

Comment:
In general, remember to include a keyword (`const temp = []` or `let temp = []`) when declaring variables, otherwise you could be reassigning a value to a global variable - e.g. there might be another `temp` variable already declared which you are unaware of.